.github/workflows/nestjs.yml

```diff
@@ -0,0 +1,36 @@
+name: Nest.js ci/cd pipeline
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+  build:
+    runs-on: self-hosted
+
+    steps:
+      - uses: actions/checkout@v3
+      - run: docker stop $(docker ps -q --filter ancestor=dev-aurora-telegram )
+      - run: docker build -t dev-aurora-telegram .
+        if: always()
+      - name: send notification to aurora group
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.telegram.org/bot${{ secrets.TELEGRAM_TOKEN }}/sendMessage?chat_id=${{ secrets.TELEGRAM_TO }}&text=[success] aurora-telegram container ready'
+          method: 'POST'
+          customHeaders: '{"Content-Type": "application/json"}'
+
+  deploy-dev:
+    if: always()
+    runs-on: self-hosted
+    needs: [build]
+    steps:
+      - run: docker run -d -p3000:3000 --restart=always dev-aurora-telegram
+      - name: send notification to aurora group
+        uses: fjogeleit/http-request-action@v1
+        with:
+          url: 'https://api.telegram.org/bot${{ secrets.TELEGRAM_TOKEN }}/sendMessage?chat_id=${{ secrets.TELEGRAM_TO }}&text=[success] aurora-telegram published to dev http://134.249.99.75:3000/'
```

Comment:
There is no message for failed deploy - please, add it through if: failure() section